### PR TITLE
Implement OpenROAD padring

### PR DESCRIPTION
**Pull request by gadfort**
Changes:
- Adds padring script access to the floorplanning step in OR
- Bumps the required version of OpenROAD (@WRansohoff can we work on getting the CI running the new version)

### CODE REVIEW

**Comment by WRansohoff** [comment]
Congrats on getting this merged into OpenROAD! I started building the new version on the CI runner, and will retry the tests when it's done.

**Comment by WRansohoff** [comment]
Update is done, looks like it is down to one test failure, on the number of warnings in `test_gcd` @gadfort

**Comment by gadfort** [comment]
> Update is done, looks like it is down to one test failure, on the number of warnings in `test_gcd` @gadfort 

Awesome. I'll fix that this evening

File: siliconcompiler/libs/sky130io.py [hunk 1]
```diff
@@ -34,6 +34,11 @@ def setup(chip):
     lib.add('output', stackup, 'gds', os.path.join(libdir, 'sky130_fd_io.gds'))
     lib.add('output', stackup, 'gds', os.path.join(libdir, 'sky130_ef_io__gpiov2_pad_wrapped.gds'))
 
+    lib.set('asic', 'cells', 'filler', ['sky130_ef_io__com_bus_slice_1um',
```
Comment (WRansohoff) [inline siliconcompiler/libs/sky130io.py:37]: Neat use of the fill cell parameter - I'm guessing these are the cells that fill the gaps between individual pads?

**Review by WRansohoff — APPROVED** [review]

File: siliconcompiler/libs/sky130io.py (hunk 1 above)
Comment (gadfort) [reply siliconcompiler/libs/sky130io.py:37]: yeah, this will only work on older nodes where the fillers are the same for the different pad orientations

**Pull request merged**